packages/repository/src/query.ts

```diff
@@ -56,7 +59,7 @@ export interface Where {
 /**
  * Order by direction
  */
-export type Direction = 'ASC' | 'DESC';
+export type Direction = 'ASC' | 'DESC' | 1 | -1;
```

Comment:
Is 1 and -1 compatible with our existing connectors? I think this is showing the problem with trying to implement a solution that supports all ORMs - juggler uses `ASC | DESC`, TypeORM uses `1 | -1`, how are we going to design a type system that allows only the valid values, depending on what ORM is used under the hood?

Reply:
IMO, we should support only `ASC` and `DESC`, and the TypeORM bridge should convert these values to whatever TypeORM uses.

Reply:
+1 on not trying to get 1 solution that supports all ORMs, you will end in with a code really difficult to maintain and unnecessary footprint.

I still think TypeORM should be an extension (not mixin) and should not be coupled in any way to the juggler. I honestly won't build on top of the Juggler, nor the REST components, I expect to cleanly install TypeORM in LoopBack.